### PR TITLE
Added the possibility of selecting a thumbnail using the seekBar

### PR DESCRIPTION
**Pull request by FerrariBruno**

### CODE REVIEW

**Review by f7deleon — APPROVED** [review]

File: k4l-video-trimmer/src/main/java/life/knowledge4/videotrimmer/K4LVideoTrimmer.java [hunk 1]
```diff
@@ -473,6 +474,14 @@ private void setProgressBarPosition(int position) {
     }
   }
 
+  /**
+   *
+   * @return Selected thumbnail position in millis
+   */
+  public int getThumbnailPositionInMillis() {
```
Comment (mirland) [inline k4l-video-trimmer/src/main/java/life/knowledge4/videotrimmer/K4LVideoTrimmer.java]: shouldn't it be a nullable Integer?

**Review by mirland — APPROVED** [review]

File: k4l-video-trimmer/src/main/java/life/knowledge4/videotrimmer/K4LVideoTrimmer.java (hunk 1 above)
Comment (FerrariBruno) [reply k4l-video-trimmer/src/main/java/life/knowledge4/videotrimmer/K4LVideoTrimmer.java]: I don't think so, I want it to be zero if the seekbar is not touched (the thumbnail would be the first frame of the video, as it was until now)

Comment (mirland) [reply k4l-video-trimmer/src/main/java/life/knowledge4/videotrimmer/K4LVideoTrimmer.java]: yes that makes sense, but it should be a decision of your app, right now it's a library decision.
never mind it's ok right now

Comment (FerrariBruno) [reply k4l-video-trimmer/src/main/java/life/knowledge4/videotrimmer/K4LVideoTrimmer.java]: yeah, you're right, I can change it to be nullable!

**Pull request merged**